frontend/device/android/powerd.lua

```diff
@@ -2,18 +2,48 @@ local BasePowerD = require("device/generic/powerd")
 local _, android = pcall(require, "android")
 
 local AndroidPowerD = BasePowerD:new{
-    fl_min = 0, fl_max = 25,
-    fl_intensity = 10,
+    fl_min = 0, fl_max = 100, -- consistant scale across android, kobo ...
```

Comment:
I don't understand why this is needed here, but if it works I see no problem.

Reply:
I changed that, because NuLuJe recommended a consistent scale across devices.
>  (I'd tend to agree, FWIW, consistent UI/settings across devices is a good thing to aim for).